central/notifiers/syslog/syslog.go

```diff
@@ -99,6 +101,17 @@ func validateSyslog(syslog *storage.Syslog) error {
 		return errors.Errorf("invalid facility %s must be between 0 and 7", facility.String())
 	}
 
+	errList := errorhelpers.NewErrorList("Key value pair validation")
+
+	for _, f := range syslog.GetExtraFields() {
+		if f.GetKey() == "" || f.GetValue() == "" {
+			errList.AddString("all extra fields must have both a key and a value")
```

Comment:
Error list like this isn't particularly useful because it's just going to be exact same string "all extra fields must have both a key and a value" repeated a bunch of times. 

My 2c: loop through the extra fields and immediately return an error (like line 101) if _any_ of them are missing key or value

Reply:
done